lessons/pythonClasses/Classes_in_Python.ipynb

```diff
@@ -0,0 +1,356 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# The Class Structure in Python\n",
+    "Adapted from https://www.jeffknupp.com/blog/2014/06/18/improve-your-python-python-classes-and-object-oriented-programming/\n",
+    "\n",
+    "## What is a Class?\n",
+    "\n",
+    "A class is a structure in Python that can be used as a blueprint to create objects that have \n",
+    "1. prototyped features, \"attributes\" that are variable\n",
+    "2. \"methods\" which are functions that can be applied to the object that is created, or rather, an instance of that class. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Defining a Class\n",
+    "\n",
+    "We want to define a class called *client* in which a new instance stores a client's name, balance, and account level.\n",
```

Comment:
PEP8 is CamelCase for classes, but more importantly, the example below is `myClass`, not `client`.